thoth/prescriptions_refresh/cli.py

```diff
@@ -177,4 +177,11 @@ def thoth_community() -> None:
         handlers.thoth_community(prescriptions)
 
 
+@cli.command("pypi-downloads")
+def package_downloads() -> None:
```

Comment:
Please rename to match the command:

```suggestion
def pypi_downloads() -> None:
```

Also, it would be great to have the handler present in a file named `pypi_downloads`.